docs/blog/20221215-confused-Techie-v1.100.1-beta.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Our First Release!
+author: confused-Techie
+date: 2022-12-15
+category:
+  - dev
+tag:
+  - sunset
+---
+
+Check out our first GitHub Release for Pulsar! [Available Now!](https://github.com/pulsar-edit/github/releases/tag/)
```

Comment:
TODO: We can change this to point to something guaranteed not to 404. Either a specific tag, or just the Releases overview page, since this first release will be the only release, once it's up. A URL ending in `/tag/` with no specific tag after will yield a "404 not found" page.

- https://github.com/pulsar-edit/pulsar/releases/tag/v1.100.0-beta (once it's up)
- Or: https://github.com/pulsar-edit/pulsar/releases (any time)

Tiny note: This URL is pointing to our pulsar-edit/**github** currently. We can change it to --> pulsar-edit/**pulsar**.

Reply:
Agree

Reply:
@DeeDeeG absolutely, tis URL is just a stand in, for once we have everything else finalized